## Message definition for Phidget encoder

Header header

int64 tick_0
int64 tick_1
int64 tick_2
int64 tick_3

float64 speed
